include/syntax/php

```diff
@@ -0,0 +1,381 @@
+// supported feature lists -----------------------------------------------------
+
+supported_apis = ["simple", "generic", "record"];
+supported_api_styles = ["free-form"];
+supported_code_models = ["loop-switch"];
+supported_targets = ["code", "dot"];
+supported_features = ["nested-ifs", "bitmaps", "tags", "captures", "captvars"];
+
+
+// language-specific options ---------------------------------------------------
+
+semicolons = 0;
+backtick_quoted_strings = 1;
+single_quoted_strings = 1;
+indentation_sensitive = 0;
+wrap_blocks_in_braces = 1;
+
```

Comment:
I've added a new configuration `special_escapes` that needs to go here, see other syntax files.